### ISSUE TITLE
[enhancement]: Embed dynamic prompt into images alongside the final prompt.

### ISSUE THREAD

**Issue by 19wolf** [invoke-ai/InvokeAI#6657]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What should this feature add?

If I'm running many iterations using a dynamic prompt ("{big|small} room with {red|yellow|blue} {walls|floors|windows}"), it's sometimes hard to recall the original parameters for a particular group of images. It would be useful to recall a dynamic prompt in some cases.

**Comment by vkleinmp** [comment]
Yes please. Same here, running many dynamic prompts, or I would run them in Invoke instead of ComfyUi if I could get the whole elaborate prompt back from the images and not only the final prompt leading to that specific image. There are more deal breaker with dynamic prompts (no real random, it must generate all variations, which sucks).
